config/config_env.go

```diff
@@ -30,13 +30,15 @@ import (
 
 const (
 	// environment variable names
-	envServiceName            = "JAEGER_SERVICE_NAME"
-	envDisabled               = "JAEGER_DISABLED"
-	envRPCMetrics             = "JAEGER_RPC_METRICS"
-	envTags                   = "JAEGER_TAGS"
-	envSamplerType            = "JAEGER_SAMPLER_TYPE"
-	envSamplerParam           = "JAEGER_SAMPLER_PARAM"
+	envServiceName  = "JAEGER_SERVICE_NAME"
+	envDisabled     = "JAEGER_DISABLED"
+	envRPCMetrics   = "JAEGER_RPC_METRICS"
+	envTags         = "JAEGER_TAGS"
+	envSamplerType  = "JAEGER_SAMPLER_TYPE"
+	envSamplerParam = "JAEGER_SAMPLER_PARAM"
+	// Deprecated: use envSamplingEndpoint instead
 	envSamplerManagerHostPort = "JAEGER_SAMPLER_MANAGER_HOST_PORT"
```

Comment:
```suggestion	
	envSamplerManagerHostPort = "JAEGER_SAMPLER_MANAGER_HOST_PORT" // Deprecated by envSamplingEndpoint
```
to preserve formatting/alignmeng